PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,9 @@
 Thanks for submitting your Pull Request!
 
-Please make sure your PR meets the following requirements:
+Please refer to our [guidelines for making contributions](https://github.com/jbosstm/narayana/blob/master/CONTRIBUTING.md) when creating your pull request. In particular, it helps the reviewer if you ensure that the:
 - [ ] Pull Request title is properly formatted: JBTM-XYZ Subject
-- [ ] Pull Request contains link to the JIRA issue(s)
+- [ ] Pull Request contains a link to the JIRA issue(s) and that they contain sufficient information for the reviewer to be able to gauge whether or not the proposed changes correctly address the issue
 
-Our style rules for submitting code are as follows:
-
-* If you add a new file it MUST adhere to our checkstyle ruleset.
-  1. If you change a file (in a non trivial way) you are allowed (MAY) to reformat the code to conform to our checkstyle ruleset (`org.jboss.narayana:checkstyle`).
-  2. If you choose not to reformat it then you SHOULD, where possible, try to follow the same style that's already in use for that file.
-
-
-The build axis can be controlled by prefixing a ! on the following as appropriate.
+The build axis can be controlled by prefixing a ! on the following as appropriate. Note that the default is not to test on windows (!NO_WIN).
```

Comment:
> [...] Note that the default is not to test on windows (!NO_WIN)

IIRC, NO_WIN means that the PR will not be tested on Windows, hence, !NO_WIN means that the PR will be tested on Windows (double negatives can be confusing). If this is the case, the above sentence should be changed with

[...] Note that the default behaviour is to test on Windows (!NO_WIN).

Maybe, to make it clearer, we could add an extra sentence saying:

[...] Note that the default behaviour is to test on Windows (!NO_WIN), therefore, if you want to disable testing the PR on Windows, use NO_WIN

Reply:
I left the text as it was before the PR.

But if you want to raise a separate one for this then go ahead, but bear in mind that I tried to keep the text minimal because most developers don't read the instructions anyway and if we add too many words there is even less chance that they will read it. In fact you would be better off changing the PROFILE to `WIN`.

Reply:
Agree :-)
Plus, (now that I think about it) the testing axis`NO_WIN` will be removed as soon as `BLACKTIE` gets deprecated.